src/main/java/Main.java

```diff
@@ -1,6 +1,100 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        // List
+        ArrayList<Product> productsName = new ArrayList<Product>();
+
+        // Переменные
+        int humans;
+        double price;
+        String products;
+        float summ = 0;
+
+        // Сканеры
+        Scanner scanner_humans = new Scanner(System.in);
+        Scanner scanner_products = new Scanner(System.in);
+        Scanner scanner_prices = new Scanner(System.in);
+
+        // Поздороваемся!
+        System.out.println("Доброго времени суток!\n" +
+                "На скольких человек необходимо разделить счёт:");
+
+        // Получаем количество людей
+        while (true) {
+            if (scanner_humans.hasNextInt()) {
+                humans = scanner_humans.nextInt();
+                while (humans <= 1) {
+                    System.out.println("Количество людей должно быть более 1");
+                    humans = scanner_humans.nextInt();
+                    if (humans > 1) break;
+                }
+                break;
+            } else {
+                System.out.println("Возможно вы ввели не число, попробуйте ещё раз!");
+                scanner_humans.nextLine();
+            }
+        }
+
+        // Получаем товары
+        while (true) {
+            System.out.println("Название товара:");
+            if (scanner_products.hasNext()) {
+                products = scanner_products.nextLine();
+                if (!products.equalsIgnoreCase("завершить")) {
+                    productsName.add(new Product(products));
+                    System.out.println("Укажите цену товара в формате рубли.копейки: ");
+                    if (scanner_prices.hasNextDouble()) {
+                        price = scanner_prices.nextDouble();
+                        while (price <= 0) {
+                            System.out.println("Значение не может быть 0 или меньше!");
+                            price = scanner_prices.nextDouble();
+                            if (price > 0) {
+                                System.out.println("Цена: " + String.format("%.2f", price));
+                                break;
+                            }
+                        }
+                        summ += price;
+                        System.out.println("Добавленные товары:");
+                        for (int i = 0; i < productsName.size(); i++) {
```

Comment:
🍏 Можно использовать цикл forEach%
```suggestion
                        for (Product name : productsName) {
```